Makefile.in.info

```diff
@@ -18,8 +18,8 @@ MAKE = make -j 8
 AR_FLAGS = rcs
 
 # Flags for debugging and regular compilation versions
-EXTRA_DEBUG_CC_FLAGS = -fPIC -g
-EXTRA_CC_FLAGS = -fPIC -O3
+EXTRA_DEBUG_CC_FLAGS = -fPIC -g -std=c++11
+EXTRA_CC_FLAGS = -fPIC -O3 -std=c++11
 
```

Comment:
We should move this out of the Makefile.in and instead append it in TACS_Common.mk, since this isn't really optional. We're already hard-coding this in the setup.py, so it makes sense to hard-code it on the c++ makefile side as well